docs/undergrad.yaml

```
@@ -9,7 +9,7 @@ Linear algebra:
     vector subspace: 'subspace'
     quotient space: 'submodule.quotient'
     sum of subspaces: 'submodule.complete_lattice'
-    direct sum: '' # this is supposed to be the internal direct sum
+    direct sum: 'direct_sum.add_submonoid_is_internal'
```

```suggestion
    direct sum: 'direct_sum.submodule_is_internal'
```
Looking at the context I suspect that this is supposed to be about vector spaces.

---

Ah, I did it this way because each docstring pointed at the stronger version. Your argument is better though.